src/software/sensor_fusion/threaded_sensor_fusion.cpp

```diff
@@ -0,0 +1,13 @@
+#include "software/sensor_fusion/threaded_sensor_fusion.h"
+
+ThreadedSensorFusion::ThreadedSensorFusion() : sensor_fusion() {}
+
+void ThreadedSensorFusion::onValueReceived(SensorMsg sensor_msg)
+{
+    sensor_fusion.updateWorld(sensor_msg);
+    std::optional<World> world = sensor_fusion.getWorld();
```

Comment:
Could this be `sensor_fusion.getWorld(sensor_msg)`? (I don't feel super strongly about it)

Reply:
:+1: changed as suggested

Reply:
ah I see that's fine